README.md

```diff
@@ -77,7 +77,7 @@ scripts/build --no-cache
 scripts/start --detach
 ```
 
-This service depends on an external Docker network named `mine-support` to communicate with other Mine Support services running alongside it. The start script will automatically create the network if it doesn't exist and the stop script will remove the network if no other containers are using it.
+This service depends on an external Docker network named `ffc-demo` to communicate with other Mine Support services running alongside it. The start script will automatically create the network if it doesn't exist and the stop script will remove the network if no other containers are using it.

```

Comment:
Still mentions "other Mine Support services"

Reply:
Actually, this bit of the Readme is getting removed in PR #10 anyway so we can leave it.